magento/pwa-studio
# Add SearchBar talons

## Description

Adds Peregrine talons for the components within Venia's `SearchBar` folder.

## Acceptance 

No net change.

### Verification Steps
1. Search for `dresses` using the search bar
2. Verify that the prompt or result message is correct in all states
3. Verify that there are no errors in the console
4. Verify that the results and links work

## Checklist
* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
